Build/scripts/pgo/pre_pgo.cmd

```diff
@@ -12,6 +12,13 @@
 :: - build (using PGO profile)
 :: - post_pgo.cmd
 
+@echo off
+
+if "%PogoConfig%"=="False" (
+    echo "---- Not a Pogo Config. Skipping step."

```

Comment:
I don't the double quotes around %PogoConfig%, the double quotes for echo is really unnecessary.

---
In reply to: [61509205](https://github.com/Microsoft/ChakraCore/pull/863#discussion_r61509205) [](ancestors = 61509205)